lib/matplotlib/image.py

```diff
@@ -854,6 +876,11 @@ class AxesImage(_ImageBase):
         'bicubic', 'spline16', 'spline36', 'hanning', 'hamming', 'hermite',
         'kaiser', 'quadric', 'catrom', 'gaussian', 'bessel', 'mitchell',
         'sinc', 'lanczos', 'blackman'.
+    interpolation_stage : {'data', 'rgba'}, default: 'data'
+        Supported values are 'data' and 'rgba'.  If 'data', interpolation
```

Comment:
Drop the supported values again, and re-wrap.
```suggestion
    interpolation_stage : {'data', 'rgba'}, default: 'data'
        If 'data', interpolation
```